server/views/pages/bookVisit/checkVisitDetails.njk

```diff
@@ -5,120 +5,133 @@
+                  text: "Change",
+                  classes: "govuk-link--no-visited-state",
+                  visuallyHiddenText: "date and time",
+                  attributes: { "data-test": "change-time" }
+                }
+              ]
+            }
           },
-          actions: {
-            items: [
-              {
-                href: paths.BOOK_VISIT.ADDITIONAL_SUPPORT,
-                text: "Change",
-                classes: "govuk-link--no-visited-state",
-                visuallyHiddenText: "additional support requests",
-                attributes: { "data-test": "change-additional-support" }
-              }
-            ]
-          }
-        }
-      ), displayRows) %}
 
-      {% set displayRows = (displayRows.push(
-        {
-          key: {
-            text: "Main contact"
+          {
+            key: {
+              text: "Additional support requests"
+            },
+            value: {
+              html: '<p data-test="additional-support">' + additionalSupport | d("None", true) | escape + "</p>"
+            },
+            actions: {
+              items: [
+                {
+                  href: paths.BOOK_VISIT.ADDITIONAL_SUPPORT,
+                  text: "Change",
+                  classes: "govuk-link--no-visited-state",
+                  visuallyHiddenText: "additional support requests",
+                  attributes: { "data-test": "change-additional-support" }
+                }
+              ]
+            }
           },
-          value: {
-            html: '<p><span data-test="main-contact-name">' + mainContactName | escape + '</span></p>
-              <p><span data-test="main-contact-number">' + mainContactNumber | d("No phone number provided") | escape + "</span></p>"
+
+          {
+            key: {
+              text: "Main contact"
+            },
+            value: {
+              html: '<p><span data-test="main-contact-name">' + mainContactName | escape + "</span></p>"
+            },
+            actions: {
+              items: [
+                {
+                  href: paths.BOOK_VISIT.MAIN_CONTACT,
+                  text: "Change",
+                  classes: "govuk-link--no-visited-state",
+                  visuallyHiddenText: "main contact",
+                  attributes: { "data-test": "change-main-contact" }
+                }
+              ]
+            }
           },
-          actions: {
-            items: [
-              {
-                href: paths.BOOK_VISIT.MAIN_CONTACT,
-                text: "Change",
-                classes: "govuk-link--no-visited-state",
-                visuallyHiddenText: "main contact",
-                attributes: { "data-test": "change-main-contact" }
-              }
-            ]
-          }
-        }
-      ), displayRows) %}
 
-      {{ govukSummaryList({
-          rows: displayRows
+          {
+            key: {
+              text: "Contact details"
```

Comment:
This is the only new section - I've just refactored the other markup for the other sections to simplify.